Tests/ConfigurationTests.swift

```diff
@@ -43,9 +43,11 @@ class ConfigurationTests: XCTestCase {
                              clientToken: "pgu-022a69d0-d651-11e5-a276-d47cee384cd5",
                              expiresOn: expiryOn,
                              graphQlUrl: "https://qamaster.aws.paylution.net/graphql",
-                             issuer: "prg-0438cadc-614c-11e5-af23-0faa28ca7c0f",
-                             restUrl: "https://qamaster.aws.paylution.net/rest/v3/",
-                             userToken: "usr-7e713de4-34e9-4e10-93cc-1f085b2d8397",
-                             authorization: nil)
+                             restUrl: "prg-0438cadc-614c-11e5-af23-0faa28ca7c0f",
+                             environment: "https://qamaster.aws.paylution.net/rest/v3/",
+                             insightsUrl: "usr-7e713de4-34e9-4e10-93cc-1f085b2d8397",
+                             issuer: "https://live.qa.paypal.com:12436/v1/tracking/batch/events/",
+                             userToken: "DEV",
```

Comment:
values are all wrong. Also use test data instead of actual prg/usr ttoken and url.

Reply:
Change the value for client token/ graphql as well.

Reply:
This is xcode's auto-fix - ha ha! Thanks for catching this. Will be careful next time